packages/lib/server/queries/teams/index.ts

```diff
@@ -54,6 +55,7 @@ export async function getTeamWithMembers(args: {
     id: true,
     name: true,
     slug: true,
+    ...(!!includeTeamLogo ? { logo: true } : {}),
```

Comment:
We need the logo for team profile, so making it possible to retrieve it under a flag to have better control